pkg/controller/command/mediatorclient/command.go

```diff
@@ -161,7 +163,10 @@ func (c *Command) Connect(rw io.Writer, req io.Reader) command.Error { //nolint:
 
 	var statusCh chan service.StateMsg
 
+	fmt.Println("####>>>> INSIDE CONNECT: RQST ###", request.StateCompleteMessageType)
```

Comment:
Are these fmt statements checked in by mistake ?

Reply:
removed them